### DISCUSSION TITLE
How to close Netflix?

### DISCUSSION THREAD

**Discussion by mspanther** [moraroy/NonSteamLaunchers-On-Steam-Deck#102] [General]
This is probably a simple answer, and I'm willing to bet it's common sense. I'm a new decker, had it about 6 months now. I just started tweaking and adding stuff beyond the stock deck. I saw this GitHub via a YouTube link, and install of the game launchers was a breeze. I however can not figure out how to exit out of Netflix. I haven't tried the other streaming apps yet, want to tackle one problem at a time if encountered. I have a cursor but there is no area that lets me close it. I'm a newbie, and expect this to be a duh moment once I figure it out.

**Comment by moraroy** [comment]
@mspanther if you have a cursor im assuming you're using a mouse or keyboard. If you are I would press esc or the "windows" button on your keyboard. Also just to let you know, you can open up "Netflix" in game mode as opposed to opening it in desktop mode. Using the "steam" button in game mode allows you to close the Netflix shortcut, it will say "Exit Game".

**Reply by mspanther** [reply]
Well that worked. Which is the exact same method I had anticipated working before the post. Guess I had a gremlin of a hiccup. I tried a few times and the deck was not responding to any input from the SD, besides the trackpad which was a mouse cursor. At least I don’t feel as dumb, just a minor weirdness along the way. Thanks, the issue is resolved.

**Reply by moraroy** [reply]
@mspanther no worries I'm glad it's working now!!
